cli/compose/loader/volume.go

```diff
@@ -70,6 +70,8 @@ func populateFieldFromBuffer(char rune, buffer []rune, volume *types.ServiceVolu
 		switch option {
 		case "ro":
 			volume.ReadOnly = true
+		case "rw":
```

Comment:
Yes, but `docker stack` uses the services API, which never supported these fields, so I don't think they would have ever worked.